src/lib/components/Codelab/index.js

```diff
@@ -47,47 +61,51 @@ class Codelab extends BaseElement {
     return container;
   }
 
-  firstUpdated() {
-    const mql = window.matchMedia("(min-width: 865px)");
-    this.toggleIframeEnabled({matches: mql.matches});
-    // Update Glitch iframe src when the user changes the window size.
-    mql.addListener(this.toggleIframeEnabled.bind(this));
-  }
-
-  toggleIframeEnabled(event) {
-    this.iframeEnabled = !!this.glitch && event.matches;
-  }
-
-  get src() {
+  glitchSrc(embed) {
     let url = `https://glitch.com/embed/?attributionHidden=true`;
 
     if (this.path) {
       url += `&path=${encodeURI(this.path)}`;
     }
 
-    url += `#!/embed/${encodeURI(this.glitch)}`;
+    if (embed) {
+      url += `#!/embed/${encodeURI(this.glitch)}`;
+    }
 
     return url;
   }
 
   render() {
-    /* eslint-disable indent */
+    const loadGlitch = !this._isMobile && this.glitch;
```

Comment:
Maybe we should switch to `isDesktop` as the positive side of things is easier to read. That will remove plenty of `!`

Reply:
Done